test/Dapper.AOT.Test/Interceptors/ExecuteBatch.output.cs

```diff
@@ -0,0 +1,256 @@
+#nullable enable
+file static class DapperGeneratedInterceptors
+{
+    [global::System.Runtime.CompilerServices.InterceptsLocationAttribute("Interceptors\\ExecuteBatch.input.cs", 14, 24)]
+    internal static int Execute0(this global::System.Data.IDbConnection cnn, string sql, object param, global::System.Data.IDbTransaction transaction, int? commandTimeout, global::System.Data.CommandType? commandType)
+    {
+        // Execute, HasParameters
+        // takes parameter: global::System.Collections.Generic.List<global::Foo.Customer>
+        global::System.Diagnostics.Debug.Assert(commandType is null);
+        global::System.Diagnostics.Debug.Assert(param is not null);
```

Comment:
in this family of cases, can we replace the `param is not null` assertion with `param is global::System.Collections.Generic.List<global::Foo.Customer>` etc?